.github/workflows/release-dataset.yml

```diff
@@ -0,0 +1,72 @@
+name: Publish dataset to production DB
```

Comment:
This code you've included here is actually to release a dataset, not publish a dataset. The difference is here all that happens is a flag `production: false` gets flipped to true so that a dataset is visible. But it doesn't upload a dataset.